game/resource/English/ability/items/tooltip_hand_of_midas.txt

```diff
@@ -1,9 +1,18 @@
 //=====================================================================
 // Hand of Midas
 //=====================================================================
+"DOTA_Tooltip_Ability_item_hand_of_midas"             "Hand of Midas"
+"DOTA_Tooltip_Ability_item_recipe_hand_of_midas"      "Hand of Midas Recipe"
+"DOTA_Tooltip_Ability_item_hand_of_midas_Description"                             "Active: Transmute - Kills a non-hero target for %bonus_gold% gold and 1.75/3/4.5x experience.  Cannot be used on Ancient Creeps."
+"DOTA_Tooltip_Ability_item_hand_of_midas_Lore"                                    "Preserved through unknown magical means, the Hand of Midas is a weapon of greed, sacrificing animals to line the owner's pockets."
+"DOTA_Tooltip_Ability_item_hand_of_midas_Note0"                               "The gold given is reliable gold (you do not get the normal creep bounty)."
+"DOTA_Tooltip_Ability_item_hand_of_midas_Note1"                               "Experience gained by using Transmute is not shared."
+"DOTA_Tooltip_Ability_item_hand_of_midas_bonus_attack_speed"                  "+$attack"
+"DOTA_Tooltip_Ability_item_hand_of_midas_transmute_cast_range_tooltip"        "TRANSMUTE CAST RANGE:"
+
 "DOTA_Tooltip_Ability_item_hand_of_midas_2"             "Hand of Midas"
 "DOTA_Tooltip_Ability_item_recipe_hand_of_midas_2"      "Hand of Midas Recipe"
-"DOTA_Tooltip_Ability_item_hand_of_midas_2_Description"                             "Active: Transmute - Kills a non-hero target for %bonus_gold% gold and %xp_multiplier%x experience.  Cannot be used on Ancient Creeps."
+"DOTA_Tooltip_Ability_item_hand_of_midas_2_Description"                             "Active: Transmute - Kills a non-hero target for %bonus_gold% gold and 1.75/3/4.5x experience.  Cannot be used on Ancient Creeps."
```

Comment:
It doesn't display the numbers correctly. 1.75 will be shown as 1.8, so this way is exactly how they solved it in the original dota 2 tooltips